### PR TITLE
No longer joining alt_identifier into our RegTAP template query.

### LINKED ISSUE #560: Stale alt_identifier join damages RegistryResource-s with multiple alternative identifiers
During our recent consideration of what to do with alternative identifiers in our registry API (PR #492) and our decision to to pull them every time (PR #523), we forgot to remove alt_indentifiers from the template query.

That, in turn, is a problem when a resource has multiple alt_identifiers; for instance,

```
from pyvo import registry

print(
    registry.search(
        registry.Ivoid("ivo://cds.vizier/iii/175")
    )[0].describe())
```

will show many identical interfaces; this is because there are four alternate identifiers on this record.

The fix is simple: drop the alt_identifiers joins.

### PR DESCRIPTION
**Pull request by msdemlei**
This fixes #560.

### CODE REVIEW

**Comment by ManonMarchand** [comment]
Ah yes, we missed it!

**Review by ManonMarchand — APPROVED** [review]

**Comment by msdemlei** [comment]
On Wed, Jun 26, 2024 at 12:25:17AM -0700, Manon Marchand wrote:
> @ManonMarchand approved this pull request.

Thanks!  I'd argue this is such a minor bugfix that a changelog entry
is not necessary; and test failures (I'll fix the datalink-related
ones in #559) are unrelated to this change, so I think this is ready
to go.

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
Thanks! Also, @ManonMarchand, please feel free to merge PRs once you approve them, unless there are some remaining discussion/todo items.

**Pull request merged**
